### PR TITLE
Add GUCvoile to Jhipster's Showcase

### PR DESCRIPTION
**Pull request by GregTres**
We did a web application based on Jhipster generator and would really like to appear in the showcase to show an exemple of Jhipster project.

### CODE REVIEW

File: pages/showcase.md [hunk 1]
```diff
@@ -155,3 +155,19 @@ If you want your application to be featured here submit a PR to the [website](ht
       </div>
    </div>
 </div>
+<div class="row">
+<div class="col-xs-12 col-sm-6 col-md-4">
```
Comment (pascalgrimaud) [inline pages/showcase.md]: minor format: indent with 4 spaces plz

File: pages/showcase.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ permalink: /showcase/
 redirect_from:
   - /showcase.html
 sitemap:
-    priority: 0.8
-    lastmod: 2014-10-18T00:00:00-00:00
+  priority: 0.8
+  lastmod: 2014-10-18T00:00:00-00:00
```
Comment (pascalgrimaud) [inline pages/showcase.md]: can you update the date here plz ?

**Comment by pascalgrimaud** [comment]
@GregTres : congrats all your team, for this beautiful showcase, really impressive ! I love your video.

Just did 2 minor comments, if you can take care of it, I'll merge your PR :)

**Comment by GregTres** [comment]
@pascalgrimaud : thank you, it was really nice and helpful to work with Jhipster.

Changes are done :)

**Comment by pascalgrimaud** [comment]
@GregTres : great ! don't hesitate to tweet this and mention `@jhipster`, it will give you more visibility !

**Pull request merged**